circuit_maintenance_parser/data.py

```diff
@@ -0,0 +1,41 @@
+"""Definition of Data classes."""
+from collections import namedtuple
+from typing import List
+
+import email
+from pydantic import BaseModel, Extra
+
+
+DataPart = namedtuple("DataPart", "type content")
```

Comment:
Totally a nitpick or matter of style, but I find a list of fields more obvious/intuitive than a space-separated string of fields when defining a namedtuple:

```suggestion
DataPart = namedtuple("DataPart", ["type", "content"])
```